common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OpenIdProviderConfigurationClient.java

```diff
@@ -54,16 +56,30 @@
     private static final ExecutorService sBackgroundExecutor = Executors.newCachedThreadPool();
     private static final Map<URL, OpenIdProviderConfiguration> sConfigCache = new HashMap<>();
 
+    private static final String NA = "";
```

Comment:
I don't see a constant being necessary for one usage -- I'm fine if you just inline this

Reply:
Addressed